linux_os/guide/system/accounts/accounts-session/accounts_user_dot_no_world_writable_programs/rule.yml

```diff
@@ -20,17 +20,23 @@ severity: medium
 
 identifiers:
     cce@rhel7: CCE-80523-4
+    cce@sle12: CCE-83099-2
 
 references:
     stigid@ol7: OL07-00-020730
     disa: CCI-000366
     srg: SRG-OS-000480-GPOS-00227
     stigid@rhel7: RHEL-07-020730
+    stigid@sle12: SLES-12-010780
 
 ocil_clause: 'files are executing world-writable programs'
 
 ocil: |-
     To verify that local initialization files do not execute world-writable programs,
     execute the following command:
+    {{% if product == "sle12" %}}
+    <pre>$ sudo find /home -perm -002 -type f -name ".[^.]*" -exec ls -ld {} \;</pre>
+    {{% else %}}
     <pre>$ sudo find /home -perm -002 -type f -exec ls -ld {} -name ".[^.]*" \;</pre>
```

Comment:
This command doesn't work for me, there is a syntax error.
Could you replace the old command with yours?

Reply:
Will do.